### PR TITLE
Pull request to merge dev branch to main branch

### PR DESCRIPTION
**Pull request by Mukaba**
Tibute page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
```
Comment (Mukaba) [inline index.html:1]: - [ ] Please use the [Microverse template](https://github.com/microverseinc/readme-template) to document your repo, I notice that the main branch already has it but it's not much descriptive and professional, so Kindly suggest adding The Microverse  README to your project.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <main id="main">
+
+   <h1>Dr. Norman Borlaug</h1>
+   <div id="img-div">
+
+    <img src="dr.png" alt="" id="img">
```
Comment (Mukaba) [inline index.html:17]: - [ ] it's best practice to add a descriptive `alt=""` inside the `img` tag. I kindly suggest you to add a description to your img. for example: `this is a doctor picture`

File: index.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    <legend id="tribute-info">This is the picture of our doctor</legend>
+    <a href="" id="tribute-link" target="_blank"></a>
+
+   </div>
+
+
+
+
```
Comment (Mukaba) [inline index.html:25]: - [ ] it's best practice to remove unwanted blank space in your code to make it easier to read , I recommend you to delete blank spaces from line 21 till 31.

**Review by Mukaba — COMMENTED** [review]
# Status: REQUIRED CHANGES ♻

**Hi @Mukaba 👋🏼,** 

Good job so far! 👏👏👏 you've accomplished the majority of the project requirement !, However, There are some issues that you still need to work on to go to the next project but you are almost there! 💪


## Highlights
- Linter is set up for this project ✔️ 
- Some linters checks were not successful ❌
- Semantic tags used ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nedjwab — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @Mukaba  👋

Your project is complete! ✔️ 🏁

Your code was very professional 💯

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉

 
## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description

- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**